### PR TITLE
remove internal iterator support to get stuff compilling, update to master

### PR DESCRIPTION
**Pull request by Gankro**

### CODE REVIEW

**Comment by Gankro** [comment]
Any objections?

**Comment by reem** [comment]
Reasonable as a temporary workaround, though it always sucks to have commented out code checked in.

File: src/blist.rs [hunk 1]
```diff
@@ -189,6 +189,7 @@ impl<T> BList<T> {
         })
     }
 
+    /* FIXME(https://github.com/rust-lang/rust/issues/21750): ICE-ICE-BABY
```
Comment (csherratt) [inline src/blist.rs]: I enjoy you sense of humour @Gankro.

**Comment by Gankro** [comment]
0.0.18 released

**Pull request merged**